src/webui/src/components/overview/Progress.tsx

```diff
@@ -98,6 +101,8 @@ class Progressed extends React.Component<ProgressProps, ProgressState> {
                 </div>
             );
         }
+
+        const bestFinalResult = (typeof bestAccuracy === 'number') ? bestAccuracy.toFixed(6) : '--'; 
```

Comment:
Accuracy might be `NaN` (which **is** a `number`) in next release.

Reply:
I'll change origin code back. And then add `// eslint-disable-line @...`